src/roiextractors/imagingextractor.py

```diff
@@ -109,6 +109,20 @@ def get_video(
         -------
         video: numpy.ndarray
             The video frames.
+
+        Notes:
+        ------
```

Comment:
pydocstyle is complaining about formatting -- this should fix it.
```suggestion
        Notes
        -----
```

Reply:
Thanks, do you test this locally usually? any advice on how to go about this?

Reply:
Yes, if you install pre-commit locally then it will check all of your local commits so that you can fix as you go.  That's how I've been using it.